index.html

```diff
@@ -41,14 +41,14 @@ <h2>Backends</h2>
 
 <p>Enchant is capable of having multiple backends loaded at once. Currently, Enchant has 6 backends:</p>
 
-<!-- FIXME: add links for all backends -->
 <ul>
-<li>Hunspell (used by many other programs, including Firefox and Thunderbird)</li>
-<li>Aspell (old Ispell replacement)</li>
-<li>Hspell (Hebrew)</li>
-<li>Zemberek (Turkish)</li>
-<li>Voikko (Finnish)</li>
-<li>AppleSpell (macOS)</li>
+<li><a href="https://nuspell.github.io/">Nuspell</a> (newest spell checker to be used by many other programs, including Firefox and Thunderbird)</li>
```

Comment:
As above, please move this paragraph under the one about Hunspell.

Reply:
https://github.com/nuspell/enchant/commit/1739563777b88c5cb0203bb33ca77d784ca06d8b#diff-eacf331f0ffc35d4b482f1d15a887d3bR46